README.md

```diff
@@ -24,9 +24,101 @@ system-under-test that is expected to cause a change to some output(s).
+Causal Testing Framework (paper citation).``
+
+The paper citation should be the Causal Testing Framework [paper](https://dl.acm.org/doi/10.1145/3607184), 
+and the software citation should contain the specific Figshare [DOI](https://orda.shef.ac.uk/articles/software/CITCOM_Software_Release/24427516) of the version used in your work.
+
+
+
+<details>
+  <summary><b>BibTeX Citations</b></summary>
+  
+  <details>
+    <summary>Paper</summary>
+    
+    ```
+    @ARTICLE{Clark_etal_2023,
+    author = {Clark, Andrew G. and Foster, Michael and Prifling, Benedikt and Walkinshaw, Neil and Hierons, Robert M.
+    and Schmidt, Volker and Turner, Robert D.},
+    title = {Testing Causality in Scientific Modelling Software},
+    year = {2023},
+    publisher = {Association for Computing Machinery},
+    url = {https://doi.org/10.1145/3607184},
+    doi = {10.1145/3607184},
+    journal = {ACM Trans. Softw. Eng. Methodol.},
+    month = {jul},
+    keywords = {Software Testing, Causal Testing, Causal Inference}
+    }
+    ```
+
+  </details>
+  
+  <details>
+    <summary>Software (example)</summary>
+    
+    ```
+    @ARTICLE{Wild2023,
+    author = {Wild, Christopher and Foster, Michael and Walkinshaw, Neil and Clark, Andrew G. and Allian, Farhad and Somers, Richard},
+    title = {CITCOM Software Release},
+    year = {2023},
+    month = {nov},
+    url = {https://orda.shef.ac.uk/articles/software/CITCOM_Software_Release/24427516},
+    doi = {10.15131/shef.data.24427516.v1}
+    }
+    ```
+  </details>
+
+</details>
+
+## How to Contribute
+
+To contribute to our work, please ensure the following:
+
+1. [Fork the repository](https://help.github.com/articles/fork-a-repo/) into your own GitHub account, and [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) it to your local machine.
+2. [Create a new branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository) in your forked repository. Give this branch an appropriate name, and create commits that describe the changes.
+3. [Push your changes](https://docs.github.com/en/get-started/using-git/pushing-commits-to-a-remote-repository) to your new branch in your remote fork, compare with `CausalTestingFramework/main`, and ensure any conflicts are resolved.
+4. Create a draft [pull request](https://docs.github.com/en/get-started/quickstart/hello-world#opening-a-pull-request) from your branch, and ensure you have [linked](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls) it to any relevant issues in your description. 
+
+We use the [pytest](https://pytest.org/en/latest/) framework as our testing suite, [pylint](https://pypi.org/project/pylint/) for our code analyser, and [black](https://pypi.org/project/black/) for our code formatting.
```

Comment:
I'm not sure what to exactly put here, but we use the built-in `unittest` module to write our tests with, and `pytest` to do our test discovery as it's more flexible. I dislike this and we should move it all to Pytest, but that would only be a nice to have thing

Reply:
@christopher-wild I can highlight this distinction but I'm not sure what that would add to this point.  How about something like:

> We use the unittest and pytest frameworks as our testing suite...
 
or

> We use `unittest` to write our tests and `pytest` for our test discovery...

Reply:
Either works, I just wouldn't want someone writing new tests in pytest 👍